phalcon/di.zep

```diff
@@ -429,4 +430,101 @@ class Di implements DiInterface
 	{
 		let self::_default = null;
 	}
+
+	/**
+	 * Loads services from a config object.
+	 */
+	protected function loadFromConfig(<Config> config)
+	{
+		var services, name, service, shared;
+
+		let services = config->toArray();
+		for name, service in services
+		{
+			let shared = isset service["shared"] ? service["shared"] : true;
```

Comment:
You should probably change the default to false. The default for set's $shared variable is false. You are forcing all items to be shared items by default when loaded from the config. This is counter to current setups.